### PR TITLE
chore(ci): Automatic release to npmjs

### PR DESCRIPTION
**Pull request by fl0rek**

### CODE REVIEW

File: .github/workflows/release-plz.yml [hunk 1]
```diff
@@ -29,8 +29,33 @@ jobs:
...
 
+      - name: Install wasm-pack
+        uses: jetli/wasm-pack-action@v0.4.0
+
       - name: Run release-plz
         uses: MarcoIeni/release-plz-action@v0.5
         env:
           GITHUB_TOKEN: ${{ secrets.RELEASE_PLZ_TOKEN }}
           CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
+      - name: Release to npm
```
Comment (fl0rek) [inline .github/workflows/release-plz.yml:40]: I ended up not using release plz outputs. If something fails during npm release, release-plz won't re-trigger npm release and this version will be stuck without release. With manual version checks between npmjs and crates.io, this is somewhat idempotent.

File: .github/workflows/release-plz.yml [hunk 2]
```diff
@@ -29,8 +29,33 @@ jobs:
...
+        env:
+          NPM_REGISTRY_TOKEN: ${{ secrets.NPM_REGISTRY_TOKEN }}
+        run: |
+          CARGO_VER=$(cargo search lumina-node-wasm | sed -e 's/lumina-node-wasm = "\(.*\)".*/\1/')
+          NPM_VER=$(npm show @fl0rek/lumina-node-wasm version)
```
Comment (zvolin) [inline .github/workflows/release-plz.yml:45]: remember to change that at the end :p

File: .github/workflows/release-plz.yml [hunk 3]
```diff
@@ -29,8 +29,33 @@ jobs:
...
+
+          wasm-pack build --target web node-wasm
+          wasm-pack publish --target web --access public node-wasm
+
+          cd node-wasm/js
+          # make sure versions are set correctly for package and wasm dependency
+          npm install -E lumina-node-wasm@$CARGO_VER
+          npm version $CARGO_VER
+          npm publish --access public
+
```
Comment (zvolin) [inline .github/workflows/release-plz.yml:61]:
```suggestion
```

**Review by zvolin — APPROVED** [review]
nice

**Review by fl0rek — COMMENTED** [review]
Approving @zvolin changes, since I cannot select approve on my PR. :gear:

**Review by oblique — APPROVED** [review]
nice

**Pull request merged**